Carla-FollowLaneTraffic/tensorflow/PilotNet/utils/processing.py

```diff
@@ -0,0 +1,303 @@
+import os
+import glob
+import cv2
+import sys
+import pandas
+import random
+
+import numpy as np
+from PIL import Image
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+from skimage.io import imread
+from sklearn import preprocessing
+from skimage.transform import resize
```

Comment:
From this list of imports, the following are not used, we should remove them:

import sys

from PIL import Image
import tensorflow as tf

from skimage.io import imread
from sklearn import preprocessing
from skimage.transform import resize